python/hsfs/feature_view.py

```diff
@@ -884,6 +1120,41 @@ def train_validation_test_split(
         """
         Get training data from feature groups.
 
+        !!! example 
+            ```python
+            X_train, X_val, X_test, y_train, y_val, y_test = feature_view.train_validation_test_split(
+                validation_size=0.3,
+                test_size=0.2
+            )
+            ```
+
+        !!! example "Time Series split"
+            ```python
+            from datetime import datetime
+            date_format = '%Y-%m-%d %H:%M:%S'
+
+            start_time_train = int(float(datetime.strptime('2017-01-01 00:00:01',date_format).timestamp()) * 1000)
+            end_time_train = int(float(datetime.strptime('2018-02-01 23:59:59',date_format).timestamp()) * 1000)
+
+            start_time_val = int(float(datetime.strptime('2018-02-02 23:59:59',date_format).timestamp()) * 1000)
+            end_time_val = int(float(datetime.strptime('2019-02-01 23:59:59',date_format).timestamp()) * 1000)
+            
+            start_time_test = int(float(datetime.strptime('2019-02-02 23:59:59',date_format).timestamp()) * 1000)
+            end_time_test = int(float(datetime.strptime('2020-02-01 23:59:59',date_format).timestamp()) * 1000)
+           
```

Comment:
we do support string, and datetime objects please use either of them. This is too complex.